R/package.R

```diff
@@ -30,6 +30,9 @@ get_sw_major_minor_version <- function(sw_version) {
 #' @return Sparkling Water version
 #' @export
 get_latest_sw_version_for_spark <- function(spark_version) {
+  Spark_Version <- NULL
```

Comment:
Why do we need to set it here? Just don't see the motivation :)

Reply:
R CMD complains that there is no reference of the variable before assignment. https://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check

Reply:
I see, thx